### PR TITLE
Feature/adding suspend resume time

### PR DESCRIPTION
**Pull request by gsps1**
JIRA : https://issues.cask.co/browse/CDAP-13424
Build : https://builds.cask.co/browse/CDAP-DUT6437-2

### CODE REVIEW

File: cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/store/AppMetadataStoreTest.java [hunk 1]
```diff
@@ -279,16 +279,19 @@ public void testInvalidStatusPersistence() throws Exception {
...
     final RunId runId6 = RunIds.generate(runIdTime.incrementAndGet());
     final ProgramRunId programRunId6 = program.run(runId6);
+    Long currentTime = System.currentTimeMillis();
     // STARTING status will be ignored if there's any existing record
     txnl.execute(() -> {
       recordProvisionAndStart(programRunId6, metadataStoreDataset);
       metadataStoreDataset.recordProgramSuspend(programRunId6,
-                                                AppFabricTestHelper.createSourceId(sourceId.incrementAndGet()));
+                                                AppFabricTestHelper.createSourceId(sourceId.incrementAndGet()),
+                                                currentTime);
```
Comment (ecfm) [inline cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/store/AppMetadataStoreTest.java:288]: even thought the time unit doesn't matter here, but should be in seconds

Comment (gsps1) [reply cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/store/AppMetadataStoreTest.java:288]: done

**Comment by ecfm** [comment]
LGTM

**Comment by gsps1** [comment]
build green, merging. https://builds.cask.co/browse/CDAP-DUT6437-8

**Pull request merged**